lighthouse-extension/test/extension.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const path = require('path');
+const assert = require('assert');
+const puppeteer = require('puppeteer');
+
+const lighthouseExtensionPath = path.resolve(__dirname, '../app');
+const config = require(path.resolve(__dirname, '../../lighthouse-core/config/default.js'));
+
+puppeteer.launch({
+  headless: false,
+  executablePath: process.env.CHROME_PATH ? process.env.CHROME_PATH : undefined,
+  args: [
+    `--disable-extensions-except=${lighthouseExtensionPath}`,
+    `--load-extension=${lighthouseExtensionPath}`,
+  ],
+}).then(browser => {
```

Comment:
Let's do async/await for this file. I'm sorry we didn't sort that out beforehand.

But IMO that'll make this guy a lot more readable.

Reply:
what about node 6 which we run our PR checks on? it doesn't support async/await right?

Reply:
we're about to drop node 6 support. :) we'll land this patch in the 3.0 side of things.